### PR TITLE
[dev] Form

### PR DESCRIPTION
**Pull request by Joanartdacosta**
solving issue: https://github.com/users/doublethink13/projects/3?pane=issue&itemId=57478902

### CODE REVIEW

File: app/src/components/form/Form.js [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+        </div>
+      </div>
+
+      <div className="ml-8 w-1/2">
+        <div>
+          <p className="text-2xl opensans pb-10">PECA O SEU ORCAMENTO</p>
+          <form
+            className="w-full"
+            id="freetour"
+            action="https://formcarry.com/s/OlAl-K2DqOh"
```
Comment (doublethink13) [inline app/src/components/form/Form.js:56]: you should use server side props and env vars for this type of information. this should not be public

Comment (Joanartdacosta) [reply app/src/components/form/Form.js]: thank you! its done

File: app/.env [hunk 2]
```diff
@@ -0,0 +1 @@
+NEXT_PUBLIC_FORM_ACTION_URL=https://formcarry.com/s/OlAl-K2DqOh
```
Comment (doublethink13) [inline app/.env:1]: this should not be public

please check https://vercel.com/docs/projects/environment-variables

Comment (Joanartdacosta) [reply app/.env:1]: okay, understood! done
the env variables were included in vercel section

Comment (doublethink13) [reply app/.env:1]: then please remove it from the code

**Review by doublethink13 — APPROVED** [review]

**Pull request merged**